.github/CODEOWNERS

```diff
@@ -0,0 +1,62 @@
+# Order is important; the last matching pattern takes the most precedence.
+
+.github            @alpenlabs/admins             # only admins should own the CODEOWNERS
+.gitignore         @alpenlabs/admins
+**/Cargo.toml      @alpenlabs/dependencies       # dependencies and metadata
```

Comment:
Won't this fail to trigger on subdirectory instances of these files, since subdirectory ownership appears later?